tests/fakeredis/poetry.lock

```diff
@@ -0,0 +1,595 @@
+# This file is automatically @generated by Poetry 1.8.3 and should not be changed by hand.
+
```

Comment:
do we have to submit it, or we should add it to .gitignore?

Reply:
Generally speaking, it should be committed, so if anyone wants to run the tests locally, they will have the exact same versions as the workflow has.
(See also [official documentation](https://python-poetry.org/docs/basic-usage/#:~:text=You%20should%20commit%20the%20poetry,of%20dependencies%20(more%20below).))